src/components/Container.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import Home from './Home'
+import Search from './Search'
+import Favorites from './Favorites'
+import { Route } from 'react-router-dom'
+
+function Container(props) {    
+    return (
+        <div>
```

Comment:
instead of adding divs onto the page you can use something call react fragments

<> </> and it will not render an extra div on the screen